central/vulnmgmt/service/testutils.go

```diff
@@ -0,0 +1,32 @@
+package service
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+)
+
+func receiveWorkloads(
```

Comment:
If it's named testutils, should make sure that it can only be used within tests by adding `testing.TB` as parameter.